decorators/decorators-test-helpers.js

```diff
@@ -26,6 +26,21 @@ function testDecoratorGetter(decoratorName, decorator, propName, getter, tester)
 			tester(TesterType);
 			QUnit.equal(ran, true, "getter ran");
 		});
+
+		QUnit.test(decoratorName + " getter decorator with class Object (without decorator)", function() {
```

Comment:
Do you have to decorate a regular getter to make it...   lazy or whatever?

Reply:
to give it special caching functionality, yes